src/main/webapp/WEB-INF/tags/instructor/course/courseEditTuneSectionPermissionsDiv.tag

```diff
@@ -6,10 +6,10 @@
 <%@ attribute name="panelIndex" required="true" %>
 <%@ attribute name="sectionRow" type="teammates.ui.template.CourseEditSectionRow" required="true" %>
 
-<div id="tuneSectionPermissionsDiv${panelIndex}ForInstructor${instructorIndex}" 
-        <c:if test="${not sectionRow.sectionSpecial}"> 
-            style="display: none;"
-        </c:if> >
+<div id="tuneSectionPermissionsDiv${panelIndex}ForInstructor${instructorIndex}" data-is-originally-displayed="${sectionRow.sectionSpecial}" 
+    <c:if test="${not sectionRow.sectionSpecial}"> 
+        style="display: none;"
+    </c:if>>
```

Comment:
Maybe write this as `</c:if> >` to be consistent with the previous style as well as the style currently used for `courseEditTuneSessionPermissionDiv.tag`